ako-crd-operator/cmd/main.go

```diff
@@ -116,6 +116,11 @@ func main() {
 	if err = (&controller.ApplicationProfileReconciler{
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
+		AviClient: aviClients.AviClient[0],
```

Comment:
This should be AviClient[1]. We should use different clients for different reconcilers?